pkg/config/monitoring.go

```diff
@@ -112,7 +112,7 @@ func (m *Monitoring) GetPrometheusRetention() string {
 }
 
 func (m *Monitoring) GetPrometheusStorageRequest() string {
-	return "50Gi"
+	return "48Gi"
```

Comment:
@david-martin Are there any implications for reducing this?

Reply:
ping @david-martin

Reply:
The amount is brought over from RHMI 1 (ping @alanmoran).
I'm unsure exactly how calculated this number is based on the stable state of metrics on a running 45d window.

However, I'd have concerns about a product that isn't installed by default affecting other products/services that are installed by default. It seems an odd thing to do.
Also, this solution seems to be ignoring the capacity impact of amq streams and should be discussed at the appropriate forum so the impact is more fully understood and planned for.